tutorials/lsrro.ipynb

```diff
@@ -0,0 +1,5428 @@
+      "2025-09-18 07:45:05 [INFO] idaes.init.fs.ROUnits[2].feed_side: Initialization Complete\n",
+      "2025-09-18 07:45:11 [INFO] idaes.init.fs.ROUnits[2]: Initialization Complete: optimal - Optimal Solution Found\n",
+      "2025-09-18 07:45:12 [INFO] idaes.init.fs.PrimaryPumps[3].control_volume: Initialization Complete\n",
+      "2025-09-18 07:45:12 [INFO] idaes.init.fs.PrimaryPumps[3]: Initialization Complete: optimal - Optimal Solution Found\n",
+      "2025-09-18 07:45:13 [INFO] idaes.init.fs.BoosterPumps[2].control_volume: Initialization Complete\n",
+      "2025-09-18 07:45:13 [INFO] idaes.init.fs.BoosterPumps[2]: Initialization Complete: optimal - Optimal Solution Found\n",
+      "2025-09-18 07:45:14 [INFO] idaes.init.fs.ROUnits[3].feed_side: Initialization Complete\n",
+      "2025-09-18 07:45:19 [INFO] idaes.init.fs.ROUnits[3]: Initialization Complete: optimal - Optimal Solution Found\n",
+      "2025-09-18 07:45:19 [INFO] idaes.init.fs.BoosterPumps[3].control_volume: Initialization Complete\n",
+      "2025-09-18 07:45:20 [INFO] idaes.init.fs.BoosterPumps[3]: Initialization Complete: optimal - Optimal Solution Found\n",
+      "WARNING: Direct failed to converge in fs.NumberOfStages iterations\n",
+      "2025-09-18 07:45:20 [INFO] idaes.init.fs.product: Initialization Complete.\n",
+      "2025-09-18 07:45:21 [INFO] idaes.init.fs.EnergyRecoveryDevices[3].control_volume: Initialization Complete\n",
+      "2025-09-18 07:45:21 [INFO] idaes.init.fs.EnergyRecoveryDevices[3]: Initialization Complete: optimal - Optimal Solution Found\n",
+      "2025-09-18 07:45:22 [INFO] idaes.init.fs.disposal: Initialization Complete.\n",
+      "INFO: Finished Sequential Decomposition in 86.26 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "lsrro.initialize(m)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 4: Simulate the LSRRO flowsheet."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Simulation Results:\n",
+      "\n",
+      "----system metrics----\n",
+      "Feed: 1.05 kg/s, 66946 ppm\n",
+      "Product: 0.349 kg/s, 751 ppm\n",
+      "Brine: 0.697 kg/s, 100084 ppm\n",
+      "Volumetric water recovery: 35.0%\n",
+      "Number of Stages: 3\n",
+      "Total Membrane Area: 183.33\n",
+      "Energy Consumption: 10.4 kWh/m3\n",
+      "Levelized cost of water: 2.41 $/m3\n",
+      "Primary Pump Capital Cost ($/m3):0.8290266209380903\n",
+      "Booster Pump Capital Cost ($/m3): 0.16734617929012008\n",
+      "ERD Capital Cost ($/m3):0.06539992139610248\n",
+      "Membrane Capital Cost ($/m3): 0.14403584004108508\n",
+      "Indirect Capital Cost ($/m3): 0.6029042808326991\n",
+      "Electricity cost ($/m3): 0.7309236776200411\n",
```

Comment:
If it’s not too much trouble, shall we round those numbers to two decimal places?